translator/calm-translator/src/main/java/org/finos/calmtranslator/controller/Translator.java

```diff
@@ -0,0 +1,35 @@
+package org.finos.calmtranslator.controller;
+
+import com.structurizr.Workspace;
+import com.structurizr.util.WorkspaceUtils;
+import org.finos.calmtranslator.calm.Core;
+import org.finos.calmtranslator.translators.C4ModelTranslator;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/translate")
+public class Translator {
+
+	private final C4ModelTranslator c4ModelTranslator;
+
+	public Translator(C4ModelTranslator c4ModelTranslator) {
+		this.c4ModelTranslator = c4ModelTranslator;
+	}
+
+	@PostMapping("/c4")
+	@ResponseStatus(HttpStatus.CREATED)
+	public String c4Translation(
+			@RequestBody Core calmModel
```

Comment:
Should this be `Core` or `Calm` (not sure by the way 😄 )

Reply:
Oh I see now that this is generated code from the JSON schema. It's an interesting one for @rocketstack-matt around naming for model generation

Reply:
Yeah, it's a bit frustrating that you can;'t override it, however, it works as it is.
You can do prefix and suffix additions, however, you can't provide a map of classes to re-name

Reply:
Yeah, it's a bit frustrating that you can;'t override it, however, it works as it is.
You can do prefix and suffix additions, however, you can't provide a map of classes to re-name

Reply:
Yeah, it's a bit frustrating that you can;'t override it, however, it works as it is.
You can do prefix and suffix additions, however, you can't provide a map of classes to re-name